### PR TITLE
Fix integration tests with real prover

### PR DESCRIPTION
**Pull request by davidnevadoc**
### Description

Some integration tests were failing when using the real prover but not when using the mock prover.
In particular, the keccak tests were the ones failing due the bug described in  issue https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1016 

### Issue Link
Solves https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1016
(Last task) Solves:  https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/966

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Contents

This PR applies the same changes made in this PR https://github.com/scroll-tech/zkevm-circuits/pull/308, plus some minor parameter adjustments.

### How Has This Been Tested?

This has been tested by running the integration tests using the mock prover. That is, with setup and genda don running:
`./run.sh --steps tests --tests circutis::real_prover`

### CODE REVIEW

File: zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ use std::{env::var, vec};
 
 pub(crate) fn get_num_rows_per_round() -> usize {
     var("KECCAK_ROWS")
-        .unwrap_or_else(|_| "5".to_string())
```
Comment (leolara) [inline zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs:14]: @davidnevadoc why this increase?

File: zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs [hunk 2]
```diff
@@ -11,7 +11,7 @@ use std::{env::var, vec};
...
+        .unwrap_or_else(|_| "12".to_string())
```
Comment (leolara) [inline zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs:14]: @davidnevadoc why this increase?

Comment (lispc) [reply zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs:14]: as we discussed with Han previously，circuit being wider is friendly for super circuit, But the more columns the more  proving cost of aggregation circuit. So we did not choose minimum possible value But a bit larger

Comment (leolara) [reply zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs:14]: Ok

**Pull request merged**